oauthproxy_test.go

```diff
@@ -156,6 +156,32 @@ func TestEncodedSlashes(t *testing.T) {
 	}
 }
 
+func TestNewReverseProxyWithTimeOut(t *testing.T) {
```

Comment:
this seems to belong to the other commit

Reply:
I can move it to the first commit.

However, I then need to invent a new argument to the NewReverseProxy() function, as otherwise, I depend on passing a pointer to the Options type to avoid adding more arguments. I can do that and then move to using the pointer only, of course.

Reply:
Actually, let me refactor the order of commits a bit there.